.gitignore

```diff
@@ -1,23 +1,28 @@
-# Compiled class file
-*.class
+*/.idea
+*.iml
 
-# Log file
-*.log
+.DS_Store
+*.bu
 
-# BlueJ files
-*.ctxt
+.m2/
 
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
+.gradle
 
-# Package Files #
-*.jar
```

Comment:
we did we remove these?

Reply:
Target and .m2 folders are already included in the ignore list, so we should be fine without the *.jar exclusion

Reply:
The file was updated yesterday to unify all .gitignore files across modules